systers_portal/templates/account/logout.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+
+{% load url from future %}
+
+{% load i18n %}
+{% block head_title %}{% trans "Sign Out" %}{% endblock %}
+{% block content %}
+  <div class="mt40"></div>
+  <div class="container">
+	<form class="login form-signin" role="form" method="post" action="{% url 'account_logout' %}">
```

Comment:
Please create 2 classes that are more appropriate to a logout template. 
Use same styles as for login, but change the naming. For example, `logout` and `form-logout`.